### PR TITLE
Fix and update documentation

### PR DESCRIPTION
**Pull request by praveenkumarct**
- Add proper links in the docs.
- Update the main ReadME file to link all the package docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
-Where a static bundle points to specific SKUs, a dynamic bundle points to specific categories and stores business rules around each category. Pricing for the dynamic bundle can be a fixed amount or change based on the bundle selections. The bundles-dynamic package contains the dynamic bundles Merchant Center custom application and the backend-starter-dynamic-bundles contains starter resources for a dynamic backend.
+An alternative option is to create a new virtual SKU that represents the bundle.
+This new SKU points to all items in the bundle. After the customer purchases the
+virtual SKU, the items are packaged and delivered. This solution targets these
+virtual or post-pack bundles.
 
+Check detailed documentation about static and dynamic bundles projects,
+
+- [Bundles-static](docs/static/index.md)
+- [Bundles-dynamic](docs/dynamic/index.md)
```
Comment (lojzatran) [inline README.md:18]: Why did you remove the parts about static and dynamic bundles?

File: docs/dynamic/index.md [hunk 2]
```diff
@@ -662,5 +655,5 @@ that additional charge is and updating the bundle's price accordingly.
 
 ## Support
 
-Contact the US Professional Services team for all support requests related to
+Please create an [issue](https://github.com/commercetools/commercetools-bundles-starter/issues) in this repository or contact for all support requests related to
```
Comment (lojzatran) [inline docs/dynamic/index.md:658]: contact who?

File: docs/static/index.md [hunk 3]
```diff
@@ -674,5 +672,5 @@ PDP implementation to display all of your commercetools products.
 
 ## Support
 
-Contact the US Professional Services team for all support requests related to
+Please create an [issue](https://github.com/commercetools/commercetools-bundles-starter/issues) in this repository or contact for all support requests related to
```
Comment (lojzatran) [inline docs/static/index.md:675]: contact who?

File: packages/platform-extension-static-bundles/docs/index.md [hunk 4]
```diff
@@ -81,7 +76,7 @@ This application's dependencies may be installed using yarn.  Simply run `yarn`
...
+While it is not required to use Terraform to manage the Product Types and Types in your commercetools project, it can certainly help.  You may initialize Terraform using `yarn terraform:init`. This will require manually installing the [commercetools Terraform provider](https://github.com/labd/terraform-provider-commercetools) as well as terraform itself.  Once initialized, you may run `yarn terraform:plan` and `yarn terraform:apply` to plan and apply any configuration changes.  Terraform will make use of the same variables in `../config/default.env`, but will also load the env vars from `config/${NODE_ENV}.env`.  If you wish to not use Terraform, you ***must*** ensure that the Product Types and Types defined in the [main.tf](terraform/main.tf) are added to your project and are up to date. The product types are also defined as JSON in the [resourceDefinitions](../resourceDefinitions) directory.
```
Comment (lojzatran) [inline packages/platform-extension-static-bundles/docs/index.md:79]: Are you going to change `Environment Variables` below as well? Would be nic if it has a format like in adyen: https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/docs/HowToRun.md#required-attributes

**Review by lojzatran — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (praveenkumarct) [reply README.md:18]: It was duplicated, Check in static and dynamic bundles docs for the removed sentences. 
So I have removed only the duplicated sentences.

File: docs/static/index.md (hunk 3 above)
Comment (praveenkumarct) [reply docs/static/index.md]: https://github.com/commercetools/commercetools-bundles-starter/pull/249/commits/c732da0c9b7846d78f050d4e68a1b63ad9bdced2

File: docs/dynamic/index.md (hunk 2 above)
Comment (praveenkumarct) [reply docs/dynamic/index.md]: https://github.com/commercetools/commercetools-bundles-starter/pull/249/commits/c732da0c9b7846d78f050d4e68a1b63ad9bdced2

**Review by lojzatran — APPROVED** [review]
LGTM 🥳
